src/services/api/api-base.js

```diff
@@ -0,0 +1,82 @@
+import axios from 'axios'
+import { isFunction } from 'lodash'
+
+let sitePrefixUrl = 'https://api.cbd.int';
+
+if(/\.cbd\.int$/i   .test(window.location.hostname)) sitePrefixUrl= 'https://api.cbd.int';
+if(/\.cbddev\.xyz$/i.test(window.location.hostname)) sitePrefixUrl= 'https://api.cbddev.xyz';
+if(/\localhost$/i   .test(window.location.hostname)) sitePrefixUrl= '/';
+
+const cache          = new Map()
+const defaultOptions = { prefixUrl: sitePrefixUrl, timeout  : 30 * 1000 }
```

Comment:
use Vue global auth to to acces token as defualt behaviour 
```suggestion
const defaultOptions = { 
   prefixUrl:  sitePrefixUrl, 
   timeout  : 30 * 1000,
   token: () => Vue.$auth.strategy.token.get()
}
```